modular-docs-manual/content/topics/concept_reusing-a-module-in-an-assembly.adoc

```diff
@@ -0,0 +1,86 @@
+[#reusing-a-module-in-an-assembly]
+
+= Reusing a Module in an Assembly
+
+In some cases, a book will need to include the same module multiple times within the same assembly.
```

Comment:
Could we dispense with the 'book' thing? Let's only speak in terms of modules and assemblies.